### ISSUE TITLE
File model conflicts with File API

### ISSUE THREAD

**Issue by annevk** [WICG/entries-api#37]
See https://github.com/w3c/FileAPI/issues/41 for a discussion. We should have a single model for files in the web platform. (I.e., what kind of fields they have and what kind of values those fields can hold.)

**Comment by inexorabletash** [comment]
Yep. (As always, note that this attempts to document existing UA behavior, not define an API we're happy with)

We should check to see what UAs currently do with this API when given native files (via drag/drop or `<input type=file>`) with names that contain "forbidden" characters (NUL, `/` or `\`) ? I expect bugs and inconsistency, which might give us room to make some web-compatible fixes.

cc: @mkruisselbrink
